doc/classes/PhysicalBone3D.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		The [PhysicalBone3D] node is a physics body that can be used to make bones in a [Skeleton3D] react to physics.
+		[b]Note:[/b] In order to detect this by raycast, [member SkeletonModifier3D.active] of the parent [PhysicalBoneSimulator3D] must be [code]true[/code] and [method get_bone_id] returns a valid id ([code]&gt;= 0[/code]).
```

Comment:
```suggestion
		[b]Note:[/b] In order to detect physical bones with raycasts, the [member SkeletonModifier3D.active] property of the parent [PhysicalBoneSimulator3D] must be [code]true[/code] and [method get_bone_id] should return a valid id ([code]&gt;= 0[/code]).
```
Just language improvement, but the last part about `get_bone_id` also seems a bit unclear. There's no documentation for `get_bone_id`, so how does a user ensure that it's valid? Can they set it themselves?

Reply:
It indicates that the property's bone_name is assigned, but for some reason "bone_name" is not bound as property, hence this description. I don't know if this is done to avoid storing it in tres to give priority to the bone_id.

Reply:
I've changed the text to make it a bit more conceptual. BTW, if we need to bind "bone_name" as a property, we should probably create a separate issue. It is possible that the bindings have been removed in the past for some reason, so an investigation is needed.